.changeset/orange-ducks-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": minor
```

Comment:
Good to have a short-summary of the API changes / usage guideline, so consumers can read the changelog to adapt if needed

Reply:
Yep, I was going to do that after we finalize and all agree on the api